guardian/support-admin-console
# Channel Management refactor

## What does this change?
The naming of some files and where the files are within the repo has got a bit difficult to read with the support admin console.

This PR:
- moves all variant related files in their own directory 
-  renames the files under the new directory to make them shorter and easier to understand. There is no actual code/functionality change within this PR.



## How to test
Check that there is no change to behaviour on the RRCP, particularly around variants.


## How can we measure success?



## Have we considered potential risks?



## Images



## Accessibility
